src/test/java/org/junit/runners/CustomBlockJUnit4ClassRunnerTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2002-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
We don't have copyright and licenses in any of the other files that I'm aware of. I worry that someone in the future will see this, think it's an error, and change it to the EPL license.

There's very little code you are copying from the Spring code. Let's just rework it so we don't need to add license info